lib/open_api/serializer.rb

```diff
@@ -2,11 +2,12 @@ module OpenApi
   module Serializer
     def as_json(arg = {})
       previous = super
-      return previous unless arg.key?(:prefixes)
+      encryption_filtered = previous.except(*self.class.try(:encrypted_columns))
```

Comment:
you can replace `previous` with `super` directly here, as the variable doesn't help for clarity anymore.